variables.tf

```diff
@@ -252,4 +252,57 @@ variable "shutdown_hours" {
   })
   default     = null
   description = "shutdown_hours object"
+}
+
+## Update Policy - update_policy ##
+variable "should_roll" {
+  type        = bool
+  default     = false
+  description = "Should the cluster be rolled for configuration updates"
+}
+## roll_config ##
+variable "conditioned_roll" {
+  type        = bool
+  default     = null
+  description = "Default: false. Spot will perform a cluster Roll in accordance with a relevant modification of the cluster’s settings. When set to true , only specific changes in the cluster’s configuration will trigger a cluster roll (such as availability_zones, max_pods_per_node, enable_node_public_ip, os_disk_size_gb, os_disk_type, os_sku, kubernetes_version, vnet_subnet_ids, pod_subnet_ids, labels, taints, tags)."
```

Comment:
default value and description doesn't match, please modify

Reply:
Updated value from null to default value.